CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-10-01
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* No new clusters can be created with [Azure AD Integration (legacy)](https://learn.microsoft.com/azure/aks/azure-ad-integration-cli). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to [AKS-managed Azure AD](https://learn.microsoft.com/azure/aks/managed-azure-ad) automatically starting from 1st Dec. 2023. We recommend updating your cluster with AKS-managed Azure AD before 1 Dec 2023. This way you can manage the API server downtime during non-business hours.
+* The [vertical pod autoscaling (VPA)](https://learn.microsoft.com/azure/aks/vertical-pod-autoscaler) add-on for AKS is now generally available. 
+
+### Release notes 
+
+* Features
+  * [H100 GPUs (accelerated compute instances)](https://learn.microsoft.com/azure/virtual-machines/nd-h100-v5-series) are now supported on AKS, with two current SKU offerings: Standard_ND96isr_v5 and Standard_ND96is_v5.
+  * Support for IP address changes for [Azure Blob NFS mounts](https://learn.microsoft.com/azure/storage/blobs/network-file-system-protocol-support-how-to#step-5-install-the-aznfs-mount-helper-package) on AKS 1.27+.
+
+* Bug Fixes 
+  * Fix for some events during an upgrade such as "Deleting node" not appearing in kubectl get events.
+  * Fix for metricsDefinition operation not exposed in Azure China.
+  * Fix for [Cluster Autoscaler](https://github.com/kubernetes/autoscaler/releases) condition where nodes that VPA pods are scheduled to could not be evicted.
```

Comment:
This is the general release page, when people click this link they will go to the latest